### DISCUSSION TITLE
Please allow searching docs

### DISCUSSION THREAD

**Discussion by VictorioBerra** [logto-io/logto#1576] [Ideas]
https://docs.logto.io/ is not searchable.

**Comment by gao-sun** [comment]
Thanks, will take a look this week.

**Reply by gao-sun** [reply]
<img src="https://user-images.githubusercontent.com/14722250/179439047-2d243fad-c796-413d-b9e4-2de24ee08673.png" width="400" />

just applied for docsearch. will add the search bar once approved.

**Reply by gao-sun** [reply]
https://github.com/logto-io/docs/pull/210 going to work once this PR merged

**Reply by gao-sun** [reply]
should work now

**Reply by VictorioBerra** [reply]
Looks great, thank you!
